install/src/Env.hs

```diff
@@ -63,9 +77,11 @@ findInstalledGhcs = do
 -- command fits to the desired version.
 getGhcPathOf :: MonadIO m => VersionNumber -> m (Maybe GhcPath)
 getGhcPathOf ghcVersion =
-  liftIO $ findExecutable ("ghc-" ++ ghcVersion) >>= \case
+  liftIO $ findExecutable ("ghc-" ++ ghcVersion <.> exe) >>= \case
     Nothing -> lookup ghcVersion <$> getGhcPaths
     path -> return path
+  where exe | isWindowsSystem = "exe"
```

Comment:
Shake already provides this function, iirc.

Reply:
Yeah, good catch, done